bedrock/products/templatetags/misc.py

```diff
@@ -117,16 +123,19 @@ def vpn_monthly_price(ctx, plan='monthly', lang=None):
         {{ vpn_monthly_price(plan='12-month') }}
     """
 
-    pricing_params = settings.VPN_VARIABLE_PRICING.get(lang, settings.VPN_VARIABLE_PRICING['us'])
-    default_amount = pricing_params[plan]['price']
-    euro_amount = settings.VPN_VARIABLE_PRICING['de'][plan]['price']
-    usd_amount = settings.VPN_VARIABLE_PRICING['us'][plan]['price']
+    cc = lang.split('-')[1].lower() if lang and '-' in lang else lang
+    default_params = settings.VPN_VARIABLE_PRICING.get(cc, settings.VPN_VARIABLE_PRICING['us'])
```

Comment:
Perhaps this should be a `get_default_params(lang)` helper function?

Reply:
Good suggestion!